main.go

```
@@ -1,7 +1,149 @@
+	}
+	return SMTPEnvs{
+		Email: email,
+		SMTP_PWD: smtp_pwd,
+		SMTP_SERVER: smtp_server,
+		SMTP_PORT: smtp_port,
+	}, nil
+}
 
 func main() {
-	fmt.Println("piszcodziennie.pl init")
+	godotenv.Load(".env")
```

golangci-lint (1.62.2)

Error return value of `godotenv.Load` is not checked

(errcheck)